todoapp_server/users/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from rest_framework import routers
+from .views import UserViewSet
+
+router = routers.SimpleRouter()
```

Comment:
sobizrom da ces imat ii router za todos, hajde ovdje da ga nazovemo jos samo `userRouter` i tako ga onda kosristis na dalje i unutar `urls` file-a

Reply:
Krenuo sam to da uradim i na kraju ga ostavio tako..